### ISSUE TITLE
请问支持fine-tune吗

### ISSUE THREAD

**Issue by lu161513** [CyberZHG/keras-bert#86]
以分类任务为目标，用当前的语料fine-tune预训练好的bert模型

**Comment by CyberZHG** [comment]
https://github.com/CyberZHG/keras-bert/blob/master/README.zh-CN.md

![image](https://user-images.githubusercontent.com/853842/59845019-e64dac80-938e-11e9-803f-1e4520f15e6f.png)

**Comment by lu161513** [comment]
> https://github.com/CyberZHG/keras-bert/blob/master/README.zh-CN.md
> 
> ![image](https://user-images.githubusercontent.com/853842/59845019-e64dac80-938e-11e9-803f-1e4520f15e6f.png)

了解，我看这个分类示例中，直接load了pretrain的模型，然后接着接上了全连接，然后softmax，这样就默认了进行fine-tune嘛？
如果我就想用默认的pretrain模型->全连接->softmax，不需要fine-tune，需要怎么做？是不是得用extract_embeddings，后面再自己做

**Comment by CyberZHG** [comment]
`load_trained_model_from_checkpoint`中参数`trainable`填`False`。
